### PR TITLE
Add release automation scripts for Node buildpacks

### PR DESCRIPTION
**Pull request by colincasey**
Adds the release automation scripts for:
* Preparing all Node.js buildpacks for release
* Publishing all Node.js buildpacks

[GUS-W-13658749](https://gus.lightning.force.com/a07EE00001UkjqiYAB)

Related: #573, https://github.com/heroku/languages-github-actions/pull/34

### CODE REVIEW

File: .github/workflows/prepare-release.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Prepare Buildpack Releases
+
+on:
+  workflow_dispatch:
```
Comment (joshwlewis) [inline .github/workflows/prepare-release.yml:4]: Have you triggered a workflow to make sure it runs correctly?

**Review by joshwlewis — APPROVED** [review]

File: .github/workflows/prepare-release.yml (hunk 1 above)
Comment (colincasey) [reply .github/workflows/prepare-release.yml:4]: it'll fail until #573 is sorted out

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,92 +1,81 @@
...
-        name: Upload buildpackage to release
-        uses: svenstaro/upload-release-action@v2
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: ${{ steps.package.outputs.package }}
-          tag: ${{ steps.package.outputs.id }}_${{ steps.package.outputs.version }}
-          overwrite: true
-      - id: prepare-pr
-        name: "Prepare PR with version bumps and CHANGELOG updates"
-        run: ./.github/scripts/release-workflow-prepare-pr.sh "${{ steps.package.outputs.id }}" "${{ steps.package.outputs.version }}" "docker://${{ steps.package.outputs.address }}"
```
Comment (edmorley) [inline .github/workflows/release.yml:81]: These scripts are now unused and can be deleted:
https://github.com/heroku/buildpacks-nodejs/tree/use_release_automation/.github/scripts

...and after deleting those I think these are also unused:

```
‎‎common/meta-build.sh‎
test/meta-buildpacks/nodejs/build.sh‎
‎test/meta-buildpacks/nodejs-function/build.sh‎
```

And this readme section can be removed:
https://github.com/heroku/buildpacks-nodejs/blob/da26d191d0631ea8fa2a8fe01c2282656b34b841/README.md?plain=1#L44-L55

Comment (colincasey) [reply .github/workflows/release.yml:81]: @edmorley the `build.sh` files in the `test/meta-*` projects needs to stick around a bit longer due to this code in [cutlass](https://github.com/heroku/cutlass/blob/main/lib/cutlass/local_buildpack.rb#L93-L116)

Comment (edmorley) [reply .github/workflows/release.yml:81]: > the `build.sh` files in the `test/meta-*` projects needs to stick around a bit longer due to this code in [cutlass](https://github.com/heroku/cutlass/blob/main/lib/cutlass/local_buildpack.rb#L93-L116)

Ah yes, true :-)

@colincasey Though I believe this bit of my review comment still applies?

> These scripts are now unused and can be deleted:
> https://github.com/heroku/buildpacks-nodejs/tree/use_release_automation/.github/scripts

Comment (colincasey) [reply .github/workflows/release.yml:81]: @edmorley looped back to cover this with #577 and #578

**Pull request merged**